conf/application.conf

```diff
@@ -98,12 +98,12 @@ microservice {
     }
 
     features {
+      default = disabled
       betaBanner = enabled
       changeErrorLink = enabled
-      default = disabled
       ead = enabled
       sfus = enabled
-      sfusSecureMessaging = enabled
+      secureMessaging = sfus
```

Comment:
Ok, got it. But a bit confusing just `sfus`. Maybe values like `useSfusInbox` or `viaSfusInbox` (or something like that), as well as `useExportsInbox` or `viaExportsInbox`. Or whatever, but still something more explicit.

Reply:
Maybe I will rename the flag's name? Do you think `secureMessagingInbox` is more explicit?

Reply:
That's fine. I would still use something more explicit for the value too. Maybe just `useSfus` or `viaSfus`.